h/migrations/versions/8fcdcefd8c6f_delete_orphaned_api_tokens.py

```diff
@@ -0,0 +1,56 @@
+"""Delete orphaned API tokens."""
+
+import logging
+
+from alembic import op
+from sqlalchemy import Column, Integer, UnicodeText, and_, delete, func, select
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+revision = "8fcdcefd8c6f"
+down_revision = "aeda42d241b3"
```

Comment:
This is gonna need to be changed to https://github.com/hypothesis/h/pull/8512/files.

I want to deploy this migration early tomorrow morning